velox/docs/functions/presto/tdigest.rst

```diff
@@ -0,0 +1,107 @@
+==================
+T-Digest Functions
+==================
+
+Velox implements algorithms for estimating rank-based metrics using `T-digest
+<https://doi.org/10.1016/j.simpa.2020.100049>`_. T-digest has `better
+performance <https://arxiv.org/abs/1902.04023>`_ in general and has
+better accuracy at the tails, often dramatically better, but may have worse
+accuracy at the median, depending on the compression factor used.
+
+T-digest was developed by Ted Dunning.
+
+Data Structures
+---------------
+
+A T-digest is a data sketch which stores approximate percentile information.
+The Velox type for this data structure is called ``tdigest``,
```

Comment:
Please, add these types to https://facebookincubator.github.io/velox/develop/types.html#presto-types